libs/community/langchain_community/document_loaders/directory.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         self,
         path: str,
         glob: str = "**/[!.]*",
+        exclude: Sequence[str] = (),
```

Comment:
This is a breaking change since it's a positional argument. It needs to be added as a named argument after the `*`

Reply:
Ah of course, totally missed that :sweat_smile: thanks.

Reply:
Made this a keyword-only argument.